### PR TITLE
test/imgtestlib: tag NullBuild jobs with "shell"

### PR DESCRIPTION
**Pull request by achilleas-k**

### CODE REVIEW

**Review by ondrejbudai — APPROVED** [review]
Thanks!

**Review by ondrejbudai — DISMISSED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -23,8 +23,10 @@ init:
 
 check-build-coverage:
   stage: verify
-  image: fedora:latest
+  extends: .terraform
   variables:
+    RUNNER: aws/fedora-40-{arch}
```
Comment (ondrejbudai) [inline .gitlab-ci.yml:28]: This doesn't work :/ Do we actually to have this parametrized?

Comment (achilleas-k) [reply .gitlab-ci.yml:28]: ugh, generally yes for the build jobs, which I copy+pasted from, but for this, we need a fixed arch.  SORRY!

**Review by ondrejbudai — APPROVED** [review]
Thanks!

**Comment by achilleas-k** [comment]
Sorry, had to amend again.  Dependency install requires `sudo` now.

**Review by ochosi — APPROVED** [review]
LGTM

**Comment by ochosi** [comment]
The `PR best practices` check failed because the PR has no description. It's possible that amending the description will not be reflected in the API unless you force-push, just as a hint.

**Pull request merged**